test/prob/discrete_range/discrete_range_cdf_log_test.hpp

```diff
@@ -0,0 +1,63 @@
+// Arguments: Ints, Ints, Ints
+#include <stan/math/prim.hpp>
+
+using stan::math::var;
+using std::numeric_limits;
+using std::vector;
+
+class AgradCdfLogDiscreteRange : public AgradCdfLogTest {
+ public:
+  void valid_values(vector<vector<double>>& parameters,
+                    vector<double>& cdf_log) {
+    vector<double> param(3);
+
```

Comment:
The boundary conditions `y = lower` and `y = upper` should be tested.  This applies to the other tests, too.

Reply:
I'm testing boundaries in prim tests.